charts/app/values-pr.yaml

```diff
@@ -35,4 +35,15 @@ frontend:
   deploymentStrategy: Recreate
   pdb:
     enabled: false # enable it in PRODUCTION for having pod disruption budget.
+  config:
+    REACT_APP_BACKEND_API: "https://nr-site-registry-dev-backend.apps.silver.devops.gov.bc.ca"
+    REACT_APP_AUTH_AUTHORITY: "https://epd-keycloak-dev.apps.silver.devops.gov.bc.ca/auth/realms/forms-flow-ai/"
+    REACT_APP_AUTH_CLIENT_ID: "site-web"
+    REACT_APP_AUTH_REDIRECT_URI: "https://nr-site-registry-dev-frontend.apps.silver.devops.gov.bc.ca"
```

Comment:
lets make this blank, since it is computed and different for each PR.

Reply:
> lets make this blank as '~', since it is computed and different for each PR.

REACT_APP_AUTH_LOGOUT_REDIRECT_URI  also refers to the frontend url for redirect, how should we take care of that ?

Reply:
yeah, good catch, for pr environment , lets make it `~`, since it changes for each PR and is computed in the configmap.yaml